lib/etcd_lib.py

```diff
@@ -1,4 +1,15 @@
-from charmhelpers.core.hookenv import network_get, unit_private_ip
+from charmhelpers.core import hookenv
+from charmhelpers.core.hookenv import config, network_get, unit_private_ip
+from charmhelpers.contrib.network.ip import is_address_in_network
+
+from jinja2 import Environment, FileSystemLoader
+from jinja2.exceptions import TemplateNotFound
```

Comment:
The `<<``>>` delimiters are necessary because the json model contains other "variables" wrapped in `{{``}}` that we want to leave as-is. the only thing that we want to replace is `datasource` so I used different delimiters. And since the charmhelpers function does not support custom delimiters (https://github.com/juju/charm-helpers/issues/367) I had to render the template directly using `jinja2`.